index.html

```diff
@@ -941,7 +549,7 @@ <h3>
           instance, the unloading time reveals how long the previous page takes
           to execute its unload handler, which could be used to infer the
           user's login status. These attacks have been mitigated by enforcing
-          the <a>same-origin check</a> algorithm when timing information
+          the [=same origin=] check algorithm when timing information
```

Comment:
Maybe "these attack are mitigated in HTML (link to algorithm) by enforcing a same origin check..."?